demo/vmdefs/available/centos6-wordpress/Vagrantfile

```diff
@@ -0,0 +1,32 @@
+VAGRANTFILE_API_VERSION = "2"
+N = 2 
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+    (1..N).each do |machine_id|
+        config.vm.define "machine#{machine_id}" do |machine|
+            machine.vm.box = "centos/6"
+            machine.vm.hostname = "machine#{machine_id}"
+ 
+            machine.vm.network :forwarded_port, host: 80, guest: 80,  auto_correct: true # website
+            machine.vm.network :forwarded_port, host: 443, guest: 443, auto_correct: true # ssl
+            machine.vm.network :forwarded_port, host: 3306, guest: 3306, auto_correct: true # mysql
+            ipaddr = "10.0.0.#{20+machine_id}"
+            machine.vm.network :private_network, ip: "10.0.0.#{20+machine_id}" 
+            machine.vm.synced_folder "./", "/var/www/html", type: "rsync", id: "vagrant", :nfs => false,
+                :mount_options => ["dmode=777", "fmode=666"]
+  
+            machine.vm.provider :libvirt do |libvirt|
+                libvirt.memory = 1024
+                libvirt.cpus = 1
+            end
+ 
+            machine.vm.provision "ansible" do |ansible|
+                ansible.limit = "all"
+                ansible.playbook = "playbook#{machine_id}.yml"
```

Comment:
I guess you can have one playbook file, by host you can select what should be executed. Am I wrong?

Reply:
yes 

---
- hosts: A
  tasks:  do 

- hosts: B
   tasks: do